src/components/Books/BookContainer.js

```diff
@@ -0,0 +1,25 @@
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import Book from './Book';
+import InputNewBook from './InputNewBook';
+
+const BookContainer = () => {
+  const [books, setBooks] = useState([]);
+
+  const AddBook = (title, author) => {
+    const newBook = {
+      id: uuidv4(),
+      title,
+      author,
+    };
+    setBooks([...books, newBook]);
+  };
+  return (
+    <div className="book-container">
+      <Book />
```

Comment:
- [ ] You should have a list of books return by the `Books` component. Currently there will always be only one book displayed. You can address that either by listing manually all your books or using the `map` function to iterate through your books state.